Description: don't use git in gemspec
 because it's checked during autopkgtest and git package
 is not installed.
Author: Sophie Brun <sophie@offensive-security.com>
Forwarded: not-needed
Date: Fri, 13 Mar 2020 16:06:48 +0100
Last-Update: 2020-03-13

--- a/lolcat.gemspec
+++ b/lolcat.gemspec
@@ -16,8 +16,8 @@
   s.add_dependency "optimist", "~> 3.0.0"
 #  s.add_dependency "manpages", "~> 0.6.1"
 
-  s.files         = `git ls-files`.split("\n")
-  s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
+  s.files         = Dir.glob('**/*')
+  s.test_files    = s.files.grep(%r{^(test|spec|features)/})
+  s.executables   = s.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 end
